src/partitioned-dbs/index.rst

```diff
@@ -130,6 +136,33 @@ the query across all ``q`` shards in the database. This mean that you do
 not have to wait for all ``q`` shards to respond, which is both
 efficient and faster.
 
+In other words, you can use partitions to help pre-process views. As another
+example, you may be recording traffic to various websites, and 95% of your
+queries are based around 'what happened in relation to website example.com'.
+By partitioning your database with the key ``example.com:uuid``, and using
+the ``/dbname/_partition/example.com`` query, you are significantly simplifying
+the work the query engine has to do.
+
+When to NOT use partitions?
+===========================
+
+If you can not split your data using a common partition, or you end up with
+a small number of partitions, it is more efficient to use seperate databases.
+Following on with the example above, if you are only monitoring 10 websites,
+it is far more efficient to have 10 databases, rather than 1 database with
+10 partitions, with 10,000,000 entries in one partition, 500 in another, and
+10,000 in all the others.
+
+As partitioning relates to which shard the data is placed into, you want to
+roughly balance the amount of data in each partition, but you should think
+of that as a design guideline when you are thinking about what to partition
+your data on.
+
+Additinally, if you are not using the ``/dbname/_partition/...`` endpoint
+for a majority of your queries, you are also potentially losing performance,
+because you won't get the automatic load-balancing of documents over shards
+that a non partitioned database gives you.
```

Comment:
Same point as the previous paragraph; besides the misspelling, this is awkwardly phrased. I like the idea of "Oh, you're not using the `_partition` specific endpoints most of the time? You probably don't need a partitioned DB."

Reply:
Possible suggestion:

> If you find that you are rarely using ``/dbname/_partition/...`` endpoints, the use of partitions may actually hurt performance. This is because CouchDB is unable to automatically distribute documents and associated queries across shards in the most efficient way.